content/includes/4-action-page-questions.html

```diff
@@ -1,4 +1,4 @@
-<div class="small-12 medium-4 columns">
+<div class="small-12 usa-width-one-third columns">
```

Comment:
@tiffanybbrown-va this still needs to be combined the `medium-4` class for the time being, right?